test/test-gotest.sh

```diff
@@ -7,29 +7,31 @@ ROOT=$(dirname "${BASH_SOURCE}")/..
 cd "${ROOT}"
 . test/util.sh
 
+tmpdir="`$mktemp --tmpdir -d tmp.XXX`"	# get a dir outside of the main package
+log="$tmpdir/$(basename $0 .sh).log"
+
 failures=''
 function run-test()
 {
 	$@ || failures=$( [ -n "$failures" ] && echo "$failures\\n$@" || echo "$@" )
 }
 
-make build
-
 base=$(go list .)
 for pkg in `go list ./... | grep -v "^${base}/vendor/" | grep -v "^${base}/examples/" | grep -v "^${base}/test/" | grep -v "^${base}/old/" | grep -v "^${base}/tmp/"`; do
-	echo "Testing: $pkg"
+	echo "  testing: $pkg"
```

Comment:
Nack. I especially dislike indentation by spaces, sorry.

Reply:
Sorry about the spaces, it's hard to get out of my system. The reason behind this was that this is the only test showing progress output during its run and I wanted to offset that from the 'suite' output.

Reply:
You are welcome to use as many tabs as you like! :)